dynmotd

```diff
@@ -16,52 +13,53 @@ PROCESS_USER=$(echo "$PROCESS"| grep -v root | awk {'print $2'} | awk '{ SUM +=
 PROCESSOR_NAME=$(grep "model name" /proc/cpuinfo | cut -d ' ' -f3- | awk {'print $0'} | head -1)
 PROCESSOR_COUNT=$(grep -ioP 'processor\t:' /proc/cpuinfo | wc -l)
 
+LOCAL_IP_ADDRESSES=$(hostname -I | sed 's/ /\n\                /g' | sed '/^[[:space:]]*$/d')
```

Comment:
you must not rely on `hostname` command which can be not available and could differ accross distros and this command seems a bit ugly for me :p 

debian 11:
```bash
$ hostname -I | sed 's/ /\n\                /g' | sed '/^[[:space:]]*$/d'
10.255.22.42
                10.45.6.108
```

archlinux:
```bash
$ hostname -I
hostname: invalid option -- 'I'
Try 'hostname --help' or 'hostname --usage' for more information.
```

In my knowledge there are 2 versions:
- the one written by Peter Tobias, part of deprecated net-tools where `-I` is available
- the one written by Debarshi Ray, par of new inetutils where only `-i` is available

It is safer to use only native tools available and equivalent for all distros

Reply:
Sure, but this role doesn't support archlinux 😅 

I'll try to find something else (less simple btw)

Reply:
but this repo and this script are not dedicated to ansible ;)